### PR TITLE
Update pd-scheduling-best-practices.md

### PR DESCRIPTION
**Pull request by lhy1024**
### What is changed, added or deleted? (Required)

Now the config description of cross-table merge is ambiguous, this pr will describe it more clearly.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: best-practices/pd-scheduling-best-practices.md [hunk 1]
```diff
@@ -258,10 +258,16 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
 
-- 创建过大量 Table 后（包括执行 `Truncate Table` 操作）又清空了。此时如果开启了 split table 特性，这些空 Region 是无法合并的，此时需要调整以下参数关闭这个特性：
-
-    - TiKV: `split-region-on-table` 设为 `false`，该参数不支持动态修改
-    - PD: `key-type` 设为 `"txn"` 或者 `"raw"`，该参数支持动态修改
+- 创建过大量表后（包括执行 `Truncate Table` 操作）又清空了。此时如果开启了 split table 特性，这些空 Region 是无法合并的，此时需要调整以下参数关闭这个特性：
+
+    - TiKV: `split-region-on-table` 设为 `false`，该参数不支持动态修改。
+    - PD: 
+       + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:265]:
```suggestion
        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 2]
```diff
@@ -258,10 +258,16 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
+       + `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
```
Comment (TomShawn) [inline best-practices/pd-scheduling-best-practices.md:266]:
```suggestion
        + `key-type` 保持 `table`，同时设置 `enable-cross-table-merge`为 `true`，该参数支持动态修改。
```

File: best-practices/pd-scheduling-best-practices.md [hunk 3]
```diff
@@ -258,10 +258,16 @@ Region Merge 速度慢也很有可能是受到 limit 配置的限制（`merge-sc
...
 
-- 创建过大量 Table 后（包括执行 `Truncate Table` 操作）又清空了。此时如果开启了 split table 特性，这些空 Region 是无法合并的，此时需要调整以下参数关闭这个特性：
-
-    - TiKV: `split-region-on-table` 设为 `false`，该参数不支持动态修改
-    - PD: `key-type` 设为 `"txn"` 或者 `"raw"`，该参数支持动态修改
+- 创建过大量表后（包括执行 `Truncate Table` 操作）又清空了。此时如果开启了 split table 特性，这些空 Region 是无法合并的，此时需要调整以下参数关闭这个特性：
+
+    - TiKV: `split-region-on-table` 设为 `false`，该参数不支持动态修改。
+    - PD: 
+        + `key-type` 设为 `txn` 或者 `raw`，该参数支持动态修改。
```
Comment (CrazyNash) [inline best-practices/pd-scheduling-best-practices.md:265]: 应该说明 是通过 pd-ctl 来修改还是pd.toml, 还是命令行参数。 我找了好久才发现是pd-ctl.

Comment (TomShawn) [reply best-practices/pd-scheduling-best-practices.md:265]: @lhy1024 PTAL

**Pull request merged**